templates/download/raspberry-pi/index.html

```diff
@@ -29,7 +29,16 @@ <h1>Install Ubuntu Server on a Raspberry Pi 2, 3 or 4</h1>
     </div>
   </div>
 </section>
-
+<section class="p-strip is-shallow notice">
```

Comment:
```suggestion
<section class="p-strip is-shallow is-bordered">
```
I think that this strip would look better with a border. Perhaps we should get a designer's opinion